.circleci/config.yml

```diff
@@ -204,6 +215,25 @@ jobs:
           name: build args
           command: |
             echo "build args: <<parameters.build_args>>"
+      - when:
+          condition: "<<parameters.release>>"
+          steps:
+            - gcp-cli/install
+      - when:
+          condition:
+            or:
+              - "<<parameters.publish>>"
+              - "<<parameters.release>>"
+          steps:
+            - gcp-oidc-authenticate
+            # TODO: also apply the below steps in the regular non-publish non-release situation?
```

Comment:
i think yes we should include this in both paths. not sure why this would be release specific, i think its mainly there to prevent dns related flakes.